build.gradle

```diff
@@ -38,12 +38,20 @@ repositories {
 dependencies {
     compile 'org.slf4j:slf4j-api:1.7.12'
 
-    def tapestry_version='5.3.7'
+    def tapestry_version='5.5.0-alpha-1'
```

Comment:
Is it really necessary do depend on the latest 5.5 release?
I don't think we use any 5.5 features unless I missed something, in which case it should be binary compatible with 5.3 branch, right?